docs/userguides/basics.md

```diff
@@ -79,36 +79,11 @@ print(cloud_usernames)
 .. _anchor_dates:
 ```
 
-## Dates
-
-py42 supports [POSIX timestamps](https://en.wikipedia.org/wiki/Unix_time) for date parameters. As an example, see
-the `departing_on_or_after_epoch` parameter in the
-`py42.sdk.clients.departing_employee.DepartingEmployeeClient.get_all()` method.
-
-```python
-import py42.sdk
-import py42.util
-from datetime import datetime, timedelta
-sdk = py42.sdk.from_local_account("https://console.us.code42.com", "my_username", "my_password")
-
-
-# Prints all the departing employee cases on or after two weeks
-
-# get a date in the future
-departing_date = datetime.utcnow() + timedelta(days=14)
```

Comment:
departure date is supposed to a str any way.

An example using posix timestamps might be more relevant somewhere else that is not basics.md